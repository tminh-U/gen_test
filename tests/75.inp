11
4 3 12 4 7 8 2 7 8 20 13 
15
8 8
9 11
3 4
6 9
11 11
8 8
7 9
1 8
11 11
8 10
8 9
1 11
10 10
7 9
7 7
