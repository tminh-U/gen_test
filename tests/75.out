2
6
3
10
2
2
8
41
2
8
4
237
2
8
2
